test/Main.test.ts

```diff
@@ -1855,6 +1918,14 @@ describe(`MainP${IMPLEMENTATION} contract`, () => {
       await expect(basketHandler.connect(owner).setPrimeBasket([token2.address], [fp('1')]))
         .to.emit(basketHandler, 'PrimeBasketSet')
         .withArgs([token2.address], [fp('1')], [ethers.utils.formatBytes32String('USD')])
+
+      await expect(basketHandler.connect(owner).forceSetPrimeBasket([token1.address], [fp('1')]))
```

Comment:
should we test these for indexBH as well?

Reply:
yes...